### ISSUE TITLE
Parsing lspci fails if physlot is a range like '0-4'

### ISSUE THREAD

**Issue by wolkenarchitekt** [kellyjonbrazil/jc#615]
I have this PCI device lspci output:
```
Slot:	c4:00.0
Class:	Ethernet controller [0200]
Vendor:	Broadcom Inc. and subsidiaries [14e4]
Device:	BCM57416 NetXtreme-E Dual-Media 10G RDMA Ethernet Controller [16d8]
SVendor:	Lenovo [17aa]
SDevice:	BCM57416 NetXtreme-E Dual-Media 10G RDMA Ethernet Controller [4048]
PhySlot:	0-4 # Fails
Rev:	01
NUMANode:	0
IOMMUGroup:	3
```
When parsing with js.parse('lspci', content), I get this error:

 ```
File "/home/ingo/src/pcimapper/.venv/lib/python3.12/site-packages/jc/parsers/lspci.py", line 162, in _process
    output[key + '_int'] = int(val, 16)
                           ^^^^^^^^^^^^
ValueError: invalid literal for int() with base 16: '0-3'

```
Parsing physlot should respect devices having more than one physical slot.
Not sure yet how this kind of value can be internally represented by jc, otherwise I would come up with a fix myself.

**Comment by wolkenarchitekt** [comment]
Maybe the `output[key + '_int']` value can just be kept empty if parsing to an int fails?
```
                try:
                    output[key + '_int'] = int(val, 16)
                except:
                    pass
```

**Comment by kellyjonbrazil** [comment]
What version of `jc` are you running? I believe this was already fixed in v1.25.3.

**Comment by wolkenarchitekt** [comment]
I'm sorry, I didn't run the most recent version. 
Thanks for the quick reply!
